Automattic/pocket-casts-ios
# [Winback] Offer Confirmation Screen

| 📘 Part of: #2445
|:---:|

Fixes #2442

This PR introduces the confirmation screen presented when the 1 month free offer is applied.

| Light | Dark | Light Contrast | Dark Contrast | Extra Dark |
| :--------: | :-------: | :-------: | :-------: | :-------: |
| ![light](https://github.com/user-attachments/assets/537cfa00-8d1f-42b2-8307-033acc7505b3) | ![dark](https://github.com/user-attachments/assets/28c00746-0f75-4df0-9073-e6bb791a50f6) | ![light-contrast](https://github.com/user-attachments/assets/4cd28eb8-6312-4f6a-8102-1c08f177235b) | ![dark-contrast](https://github.com/user-attachments/assets/1cde255e-fd9a-42a7-a778-74611f28ab15) | ![extra-dark](https://github.com/user-attachments/assets/41f1c8e1-79c1-46ec-857f-10b4bdeb895d) |
| Classic | Electricity | Indigo | Rosé | Radiation |
| ![classic](https://github.com/user-attachments/assets/5a75a7bc-9a06-45aa-80e5-aa30ea632e9c) | ![electricity](https://github.com/user-attachments/assets/8d45db78-71d9-43f0-9f00-b3084f6f5bd6) | ![indigo](https://github.com/user-attachments/assets/e11873c7-5956-489f-8f69-a206f1319c6c) | ![rose](https://github.com/user-attachments/assets/10659711-8227-40b0-a1ce-7ba227c66d61) | ![radiation](https://github.com/user-attachments/assets/f71ce3ab-47eb-4ccb-8cde-70ed72f9b5e6) |

## To test

- CI must be 🟢 
- Make sure you have the `winback` FF enabled
- Login with a free user and give to you a plus plan
- Navigate to Account - > Cancel Subscription
- Tap on Claim Offer
- You should see the Confirmation screen prompted

## Checklist
